{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/utils/tests.py

```diff
@@ -37,4 +37,4 @@ def test_get_site_url_negative(settings, custom_settings):
 def test_password_reset_email_link(user):
     context = user.reset_password_context()
     html_body = get_html_body("registration/password_reset.html", context)
-    assert f"{ context['site_url'] }/password/reset/confirm/{ context['user'].id }/{ context['token'] }" in html_body
+    assert f"{context['site_url']}/password/reset/confirm/{context['user'].id}/{context['token']}" in html_body
```

Comment:
Not sure how this wasn't failing linting before — possibly due to the flake8 upgrade.

Reply:
I think the issue here is that flake8 is pinned at the bootstrapper level Pipfile, but for generated projects it's on `*`. The upgrade of python must have triggered a newer version of flake8 that we didn't have access to with python 3.10.